update-data.sh

```diff
@@ -39,6 +42,24 @@ for VER in ${DRIVER_VERSIONS}; do
             NVIDIA_ARCH=x86_64
         fi
 
+        if [ ${ARCH} == aarch64 ]; then
+            if [ ${MAJOR_VER} -lt 470 ]; then
+                continue
+            elif [[ ${TESLA_VERSIONS} == *${VER}* ]]; then
```

Comment:
It appears that there are some Tesla drivers that exist for `aarch64`.
525.60.13: https://www.nvidia.com/download/driverResults.aspx/196882/en-us/
510.73.08: https://www.nvidia.com/download/driverResults.aspx/187948/en-us/
470.82.01: https://www.nvidia.com/download/driverResults.aspx/182615/en-us/

I don't think these should be included if they are unable to be properly tested, though.

Reply:
Only some Tesla drivers have been released for aarch64, unlike the production drivers that match all x86_64 versions since the 450 series. My PR is only for the Production drivers.